using-mobile-extensions/adobe-experience-platform-messaging/README.md

```diff
@@ -0,0 +1,211 @@
+# Adobe Experience Platform Messaging
+
+The Adobe Experience Platform Messaging mobile extension allows you to send push notification tokens and push notification click-through feedback to the Adobe Experience Platform.
+
+{% hint style="warning" %}
+The Adobe Experience Platform Messaging extension is currently in Beta development.
+{% endhint %}
+
+## Prerequisites
+
+* IMS organization is enabled for messaging.
+* [Mobile Core](../mobile-core/README.md) and [Adobe Experience Platform Edge Network](../adobe-edge/README.md) extensions have been implemented in the app.
+
+## Configure Profile Dataset in Edge Configuration
+1. Follow the [edge configuration](../../getting-started/edge-configuration.md) document to configure edge.
+2. In the screen below update the **Profile Dataset** with the `CJM Push Profile Dataset`
```

Comment:
Will all orgs provisioned for messaging have this dataset created by default or will they have to create the schema and dataset?

Reply:
All orgs will have this provisioned by cjm for them